source/task-item.js

```diff
@@ -467,10 +466,9 @@ class TaskItem extends HTMLElement {
    * Method for the styles sheets
    * @returns {string} The style sheet for the task item's shadow DOM.
    */
-  static styleSheets() {
-    return `<link rel="stylesheet" href="task.css"/>
-          <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet" />
-          <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-giJF6kkoqNQ00vy+HMDP7azOuL0xtbfIcaT9wjKHr8RbDVddVHyTfAAsrekwKmP1" crossorigin="anonymous"/>`;
+  static getStyleSheets() {
```

Comment:
[Use `document.stylesheets`](https://developer.mozilla.org/en-US/docs/Web/API/Document/styleSheets)
or also realized that maybe doing something like `document.head.getElementsByTagName('link')`  or `document.head.querySelectorAll('link')` would be easier. Doing it this way would just make it more general so this doesn't need to be manually updated in the future

Reply:
Gotcha, originally I think this was a way for them to get a string of what the style sheets should be for initial creation, but now it dynamically creates the link element, applies the correct attributes, and appends it as a child to the shadow root so it should be good now.